src/main/daml/Daml/Finance/Interface/Lifecycle/Event/Time.daml

```diff
@@ -0,0 +1,38 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Interface.Lifecycle.Event.Time where
+
+import Daml.Finance.Interface.Lifecycle.Event qualified as Base (I)
+
+-- | Type synonym for `Event`.
+type I = Event
+
+-- | Type synonym for `View`.
+type V = View
+
+-- | View for `Event`.
+data View = View {}
+  deriving (Eq, Show)
+
+-- | Event interface for the replacement of units of an instrument with a basket of other instruments (e.g. stock merger).
+interface Event where
+  viewtype V
+    -- ^ View type of interface.
+  asEvent : Base.I
+    -- ^ Conversion to base `Event` interface.
+
+  nonconsuming choice GetView : View
+    -- ^ Retrieves the instrument view.
+    with
+      viewer : Party
+        -- ^ The party retrieving the view.
+    controller viewer
+    do
+      pure $ view this
+
+-- | Type constraint for requiring templates to implement `Event`.
+type Implementation t = HasToInterface t Event
```

Comment:
Should this `Time` event interface require the base `Event` interface to be implemented?

Ie :

```suggestion
type Implementation t = (HasToInterface t Event, Event.Implementation t) 
```

Note - this suggestion requires `Implementation` to be imported.